Automattic/jetpack
# Jetpack: do not convert core/embed to videopress/video on-the-fly

This PR rollbacks with the idea to convert `core/embed` VideoPress video variation to `videopress/block` on the fly in Simple sites. Instead, we're going to add a video transform from/to.

Part of  https://github.com/Automattic/jetpack/issues/27941

#### Changes proposed in this Pull Request:

* Jetpack: do not convert core/embed to videopress/video on-the-fly

#### Other information:


#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?



#### Testing instructions:






* Go to a Simple testing site
* Add a core/embed video block
* Use a VideoPress video URL: `https://videopress.com/v/ezoR6kzb`
* Confirm the app does not transform the block on the fly
